src/css/components/menu/_menu-inline.scss

```diff
@@ -44,12 +44,14 @@
 }
 
 // Hover state
-.video-js .vjs-menu-button-inline:hover {
+.video-js .vjs-menu-button-inline:hover,
+.video-js .vjs-menu-button-inline.vjs-slider-active {
```

Comment:
The menu button isn't actually in focus here, the internal slider is. I have the slider firing an event when it's active so the menu button can catch it add this class to itself.